mpt_extension_sdk/observability/tracing.py

```
@@ -0,0 +1,70 @@
+from contextlib import contextmanager
+from typing import Any
+
+from opentelemetry.trace import Span, SpanKind, get_tracer
+
+TRACER = get_tracer("mpt_extension_sdk")
+type AttributeValue = str | int | float | bool
+type Attributes = dict[str, AttributeValue]
+
+
+@contextmanager
+def start_event_span(path: str, *, task_based: bool, event: Any) -> Iterator[Span]:  # noqa: WPS210
```

Ruff (0.15.10)

[warning] Invalid rule code in `# noqa`: WPS210

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)